pkgs/applications/networking/cluster/helm/plugins/helm-cm-push.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "helm-cm-push";
+  version = "0.10.3";
+
+  src = fetchFromGitHub {
+    owner = "chartmuseum";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-trTCDlwwru4wzFh4KW+sPqaU5ckcnmhG3wBZ4Ni1B8Y=";
+    leaveDotGit = true;
+    postFetch = ''
+      cd $out
+      git rev-parse HEAD > $out/COMMIT
+      find "$out" -name .git -print0 | xargs -0 rm -rf
+    '';
```

Comment:
I think it's fine if we just set `-X main.Revision=v${version}` in `ldflags`, instead of doing the exact commit ID extraction dance.

Reply:
Apparently, these flags are not even required because they are not used anywhere in the program.